### ISSUE TITLE
darknet.py output understanding

### ISSUE THREAD

**Issue by RJVisee44** [AlexeyAB/darknet#1553]
Hello all, 

Using darknet.py we get 

> list of tuples like
        ('obj_label', confidence, (bounding_box_x_px, bounding_box_y_px, bounding_box_width_px, bounding_box_height_px))
        The X and Y coordinates are from the center of the bounding box. Subtract half the width or height to get the lower corner.> 

I can't seem to understand how we can convert this to xmin, ymin, xmax, ymax (exact bounding box coordinates). Please help!

**Comment by RyanCodes44** [comment]
Actually, Think I get it now... 

xmin = bounding_box_x_px - (bounding_box_width_px/2)
ymin = bounding_box_y_px - (bounding_box_height_px/2)
xmax = xmin + bounding_box_width_px
ymax = ymin + bounding_box_height_px

**Comment by PeterQuinn925** [comment]
Yes. If you want an example, you could look at mine - https://github.com/PeterQuinn925/Squirrel/blob/master/sort_photos.py

**Comment by RyanCodes44** [comment]
@PeterQuinn925 Thank you. Random questions, have you ever worked with the ALOV300++ Dataset (http://alov300pp.joomlafree.it/) for tracking. I cannot for the life of me understand how to interpret their ground truth annotations.
